docs/search/extensibility/create_custom_aggregation.md

```diff
@@ -62,15 +62,15 @@ The example below uses `RangeAggregationVisitor`:
 === "Solr"
 
     ``` yaml
-    App\Query\Aggregation\Solr\PriorityAggregationVisitor:
+    app.search.solr.query.aggregation_visitor.priority_range_aggregation:
         class: Ibexa\Solr\Query\Common\AggregationVisitor\RangeAggregationVisitor
         factory: [ '@Ibexa\Solr\Query\Common\AggregationVisitor\Factory\ContentFieldAggregationVisitorFactory', 'createRangeAggregationVisitor' ]
         arguments:
             $aggregationClass: 'App\Query\Aggregation\PriorityRangeAggregation'
```

Comment:
I tested again, but I am still getting some errors, sorry I missed this yesterday
```suggestion
            $aggregationClass: 'App\Query\Aggregation\Solr\PriorityRangeAggregation'
```